src/angular/11-declarative-state/declarative-state.md

```diff
@@ -45,193 +45,179 @@ everything about how the state is created. In imperative code you'd need to read
-Forms API:
-- `displayedCity` which sets the city select control value to user input if there's been any, or clears it if the 
-   state select control has been changed
-- `stateSelectDisabled` which disables the state select control if loading is ongoing
-- `citySelectDisabled` which disables the city select control if loading is ongoing or no values are available
+- `enableStateSelect$` which enables the state select control once state values are available
+- `toggleCitySelect$` which enables/disables the city select control if loading is ongoing or no values are available
+- `clearCityWhenStateChanges$` which clears the city select control value if the state select control has been changed
 
 ## How to Verify Your Solution is Correct
 
-✏️ Update the spec file __src/app/restaurant/restaurant.component.spec.ts__ to be:
+✏️ Update the spec file **src/app/restaurant/restaurant.component.spec.ts** to be:
 
 @sourceref ./restaurant.component.spec.ts
-@highlight 3-4, 9, 136, 175-176, 189-190, 197, 202-207, 304, 307, 309, 311-313, 318, 321, 323-324, 329, 332, 335-336, 343, 351, 356, 361-363, 374, 383, 389, 394-396, 398, 407, 414, 417-418, 423-424, 426, 428, 429-438, 441-446, 448, 450-451, 453, 455, 460, 463-468, 471, 479-481, 483-484, 486-487, 489, only
+@highlight 9, 11-13, 198-203, 306, 311-314, 319, 324-325, 357-362, 372, 390-395, 405, 421-427, 442-446, 450, 453, only
 
 > If you've implemented the solution correctly, when you run `npm run test` all tests will pass!
 
 ## What You Need to Know
 
+- Creating streams of Form Control values with `valueChanges` member (you previously learned this in the [learn-angular/form-value-changes#listening-to-form-changes Filter Cities by State] section! ✔️)
 - How to perform common RxJS operations like:
-    - setting the initial value to be emitted
-    - transforming a value emitted
-    - conditionally emit values into a stream from another stream
-    - merge streams into a single stream of values 
-    - merge streams into a stream of arrays with values from each input stream
-    - use a BehaviorSubject to capture the state of a form element
-    - multicasting emissions of a "cold" observable and handle late subscribers
- 
- You've learnt all of the above as part of the earlier sections on this page! Completing the 
- [Bitovi Academy's RxJS training](../learn-rxjs.html) will help however. 
+
+  - setting the initial value to be emitted
+  - transforming a value emitted
+  - combine previous and current values of a stream as an array
+  - conditionally emit values into a stream from another stream
+  - merge streams into a stream of arrays with values from each input stream
+  - multicasting emissions of a "cold" observable and handle late subscribers
+
+    You've learnt all of the above as part of the earlier sections on this page! Completing the
+    [Bitovi Academy's RxJS training](/learn-rxjs.html) will help however.
```

Comment:
This change seems to break the url. Also there's another instance where it shows `../learn-rxjs.html`. In production, these links seem to be working. Should we revert? @fabioemoutinho

Reply:
https://www.bitovi.com/academy/learn-angular/declarative-state.html

Search "Note: You should complete" and "Completing the"

Reply:
I don't think this is part of this PR 🤔

Reply:
after internal discussion, we're going to revert code change to url for now and see if it stays working in production